x-pack/test/api_integration/apis/cloud_security_posture/update_rules_config.ts

```diff
@@ -5,83 +5,87 @@
-        .set('kbn-xsrf', 'xxxx')
-        .send({
-          name: 'Test policy',
-          namespace: 'default',
-        });
-      agentPolicyId = agentPolicyResponse.item.id;
+      await esArchiver.load('x-pack/test/functional/es_archives/fleet/agents');
     });
 
     after(async () => {
-      await kibanaServer.savedObjects.cleanStandardList();
       await esArchiver.unload('x-pack/test/functional/es_archives/fleet/empty_fleet_server');
+      await esArchiver.unload('x-pack/test/functional/es_archives/fleet/agents');
     });
 
-    it('returns a 404 response when package policy id does not exist', async () => {
-      const packagePolicyId = chance.guid();
+    it('updates package policy vars with new rules configuration', async () => {
+      const packagePolicy = await installIntegration(supertest);
+      const currentRules = await getCspRulesSO(supertest);
 
-      const { body: response } = await supertest
-        .post(`/internal/cloud_security_posture/update_rules_config`)
-        .set('kbn-xsrf', 'xxxx')
-        .send({
-          package_policy_id: packagePolicyId,
-          rules: [],
-        })
-        .expect(404);
-
-      expect(response.error).to.be('Not Found');
-    });
+      const ruleToDisable = currentRules.saved_objects[0];
+      const ruleToEnable = currentRules.saved_objects[1];
 
-    it(`Should return 200 for existing package policy id`, async () => {
-      const { body: postPackageResponse } = await supertest
-        .post(`/api/fleet/package_policies`)
+      const { body: updatedPackagePolicy } = await supertest
+        .post(UPDATE_RULES_CONFIG_ROUTE_PATH)
         .set('kbn-xsrf', 'xxxx')
         .send({
-          force: true,
-          name: 'cloud_security_posture-1',
-          description: '',
-          namespace: 'default',
-          policy_id: agentPolicyId,
-          enabled: true,
-          inputs: [],
-          package: {
-            name: 'cloud_security_posture',
-            title: 'Kubernetes Security Posture Management',
-            version: '0.0.26', // TODO: Find a method of maintaining the most recent version of the package
-          },
+          package_policy_id: packagePolicy.id,
+          rules: [
+            { id: ruleToDisable.id, enabled: false },
+            { id: ruleToEnable.id, enabled: true },
+          ],
         })
         .expect(200);
 
-      const { body: res } = await supertest
-        .post(`/internal/cloud_security_posture/update_rules_config`)
-        .set('kbn-xsrf', 'xxxx')
-        .send({
-          rules: [],
-          package_policy_id: postPackageResponse.item.id,
-        })
-        .expect(200);
+      const runtimeCfgValue = updatedPackagePolicy.vars.runtimeCfg.value;
 
-      expect(res.name).to.be('cloud_security_posture-1');
+      expect(runtimeCfgValue).to.contain(ruleToEnable.attributes.metadata.rego_rule_id);
+      expect(runtimeCfgValue).to.not.contain(ruleToDisable.attributes.metadata.rego_rule_id);
```

Comment:
our current format sends only `activated_rules` so disabled rules should not be contained.